.gitignore

```diff
@@ -24,3 +24,4 @@ yarn.lock
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.history
```

Comment:
to ensure that the directory is not included in commit change this line to `.history/*`

Reply:
Added the suggestion below.